command/kv_put.go

```diff
@@ -97,14 +97,19 @@ func (c *KVPutCommand) Run(args []string) int {
 		stdin = c.testStdin
 	}
 
-	if len(args) < 1 {
+	switch {
+	case len(args) < 1:
 		c.UI.Error(fmt.Sprintf("Not enough arguments (expected 1, got %d)", len(args)))
```

Comment:
This "expected 1" isn't true anymore, because if they provide 1 they hit the next error case.

Reply:
Good point!